infer/tests/codetoanalyze/c/frontend/atomic_expr/arithmetic.c

```diff
@@ -0,0 +1,53 @@
+#include <stdatomic.h>
```

Comment:
Could you add copyright headers to each new file (except the generated .dot files)?

```
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```